### ISSUE TITLE
Command too Long

### ISSUE THREAD

**Issue by mcroni** [giampaolo/pyftpdlib#468]
When i connect to the my pyftpdblib with a lab machine, it connect alright but gives this error in the console when i try to upload a file `command too long`

**Comment by giampaolo** [comment]
That means the client attempted to send a long command (> 2048 characters) over the command channel and the server prevented that from happening. 

Why: the server reads from the command channel in chunks, appending them to a buffer until `"\r\n"`(end of line) is encountered. A malicious client may decide to keep sending data over the command channel without ever sending the EOL marker, so the server would keep appending data to the buffer until it eventually runs out of physical memory (DoS). pyftpdlib prevents that by rejecting the command line if length > 2048.

In summary: you should check why the client is sending such a long command.
